src/fetch_real_estate_prices.py

```diff
@@ -0,0 +1,126 @@
+import json
+import numpy as np
+import requests
+import os
+import pandas as pd
+from geopy.geocoders import GoogleV3
+
+HTTP_HEADERS = {"User-Agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/32.0.1700.102 Safari/537.36"}
+OUTPUT_FILEPATH = os.path.join('data', 'real_estates_prices.csv')
```

Comment:
Your PR's made me realize we have been using a kind of standard for the `/data` that was undocumented. We are prefixing all files with date (`YYYY-MM-DD-my-dataset.xz`) and we are using `.xz` ([lzma](https://docs.python.org/3/library/lzma.html)) compression.

Would you mind updating the naming and format of the output file?